awscli/data/cli.json

```diff
@@ -58,7 +58,12 @@
         "cli-read-timeout": {
             "dest": "read_timeout",
             "type": "int",
-            "help": "<p>The maximum socket read time in seconds.</p>"
+            "help": "<p>The maximum socket read time in seconds. If the value is set to 0, the socket read will be blocking and not timeout.</p>"
```

Comment:
Where do you want me to add that? I have "...the socket read will be blocking and not timeout." at the very end for each